### PR TITLE
Add try-except to avoid Flood

### PR DESCRIPTION
**Pull request by danielgpz**
Here, I code the try-except solition, maybe isnt the solution, but you not loss anything if you try out

### CODE REVIEW

**Comment by alessiocelentano** [comment]
This is the error you talk about:
`pyrogram.errors.exceptions.bad_request_400.MessageIdInvalid: [400 MESSAGE_ID_INVALID]: The message id is invalid (caused by "messages.EditInlineBotMessage")`

This problem seems to be related to [Telegram Data Centers](https://docs.pyrogram.org/faq#what-are-the-ip-addresses-of-telegram-data-centers), and it's impossible edit an inline message in a group with a `dc_id` different than the bot one with Pyrogram (for now). The only other way to send a message without edit it is have only one query.

e.g.: `@rotogrambot eon` (it returns Empoleon, Umbreon, etc..., doesn't work)
`@rotogrambot pika`(it returns only Pikachu, works)
`@rotogrambot charizard`(it returns only Charizard, works)

TL;DR: if your chat has a different `dc_id` than the bot, you should type only full Pokemon names (or abbreviations if there's only one mon with that snippet in the name). I'll fix this bug in these days.
